tests/test_core.py

```diff
@@ -135,3 +136,17 @@ def test_simple_record_from_xml():
     expected = {'main_entry_personal_name': {'personal_name': 'Donges, Jonathan F'}}
 
     assert data == expected
+
+
+def test_none_value():
+    """Test none value."""
+    overdo = dojson.Overdo()
+    from dojson.utils import ignore_value
+
+    @overdo.over('024', '^024..')
+    @ignore_value
+    def match(self, key, value):
+        return None
+
```

Comment:
can you also add a test when the decorator is used but the return value is not None?